examples/generalized_linear_models/GLM-linear.ipynb

```diff
@@ -103,9 +103,6 @@
    "metadata": {},
```

Comment:
<p>Can you also add the versions of theano and xarray? I added how to in the last point at https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
xarray is not being used in this particular notebook so would it be okay to add to it in metadata?

Reply:
It looks like it's not being used but it is being used by ArviZ under the hood, like pymc3 uses theano under the hood even if we don't import or call it explicitly.